deploy/olm-catalog/opendatahub/1.4.1/manifests/opendatahub-operator.v1.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,289 @@
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-notebook-controller"
+                  }
+                },
+                "name": "odh-notebook-controller"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "odh-model-controller"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "model-mesh"
+                  }
+                },
+                "name": "model-mesh"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "metadata-store-mariadb",
+                    "ds-pipeline-ui",
+                    "object-store-minio",
+                    "default-configs"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "data-science-pipelines"
+                  }
+                },
+                "name": "ds-pipelines"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "grafana/cluster"
+                  }
+                },
+                "name": "grafana-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "grafana/grafana"
+                  }
+                },
+                "name": "grafana-instance"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/cluster"
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              }
+            ],
+            "repos": [
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v1.4"
```

Comment:
I guess we would have another PR for updating the olm-catalog for 1.4.1 release. But just to note here, this uri needs to be updated

Reply:
For the 1.4 releases, `v1.4` is a floating tag that will get updated to point to the latest `1.4.z` release